ibis/backends/base/sql/__init__.py

```diff
@@ -200,7 +200,9 @@ def _register_in_memory_table(self, table_op):
 
     def _register_in_memory_tables(self, expr):
         if self.compiler.cheap_in_memory_tables:
-            for memtable in lin.traverse(_find_memtables, expr.op()):
+            for memtable in graph.traverse(
+                _find_memtables, expr.op(), types=ops.Node
```

Comment:
Can we remove `_find_memtables` or replace it with `lambda op: (graph.proceed, op)` and pass `types=ops.InMemoryTable`?